NewForce-at-Mountwest/bangazon-site-pig-pong
# categories index ticket 11

##Description
Able to view categories and product count per categories and 3 products in each category below each category name that are links to details pages.

##Related Ticket(s)
11
##Steps to Test


Clone down BangazonSite/pig-pong git repository in Git Bash
CD into bangazon/pig-pong
CD into BangazonSite.sln
Run 'Start_BangazonSite.sln"

When Visual Studio opens up, you'll see a Green 'Play' button at the top of the program
Click that button and it will open a webpage of the BangazonSite project
Click Categories at the top in navbar
Should see list of categories with number counts and a few instances as link below each category
Click link and you'll be taken to product details page.

Happy Testing!

Checklist:
[x] I have commented my code, particularly in hard-to-understand areas
[ ]I have made corresponding changes to the documentation
[ ]My changes generate no new errors
[ ]I have added tests that prove my fix is effective or that my feature works
[ ]New and existing unit tests pass locally with my changes
